gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/lambda/GValueConstantTraversal.java

```diff
@@ -57,6 +57,17 @@ public boolean equals(final Object other) {
         return constantTraversal.equals(other);
     }
 
+    @Override
+    public GValueConstantTraversal<S, E> clone() {
+        GValueConstantTraversal<S, E> clone = new GValueConstantTraversal<>(GValue.of(this.end.getName(), this.end.get()));
+        try {
+            clone.setGValueManager(this.getGValueManager().clone());
+        } catch (CloneNotSupportedException e) { //TODO:: handle properly
```

Comment:
There needs to be a revisit for cloning of GValues, GValueManager, and StepPlaceholders. I consider this out of scope for this PR and intend to address this soon.

Reply:
I'm still leaving this scope to a followup on cloning, although `AbstractLambdaTraversal.clone()` does not throw `CloneNotSupportedException` so adding it to the try does not have any effect. There's actually no way for `GValue` or `GValueManager` to throw `CloneNotSupportedException` either given the current implementation, so in the followup this will be corrected and the try-catch will go away altogether.